### DISCUSSION TITLE
Avoid returning "paragraphs" and "words" in API JSON response?

### DISCUSSION THREAD

**Discussion by lvca** [deepgram/community#809] [General help] [answered]
With long transcripts, the content in "words" and "paragraphs" in the JSON response is 90% of the response. Since I'm not using it, is there a way to tell the API not to return it? It should benefit latency and bandwidth + CPU on Deepgram servers.

Thanks,
Luca

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by nikolawhallon** [comment, marked as answer]
words have to be constructed, as that's what the transcript is constructed from, but paragraphs are an optional feature, what query parameters are you specifying? by default paragraphs are not returned, but I believe if smart_format is turned on, this will return paragraphs

(you can always ignore these fields and just grab the transcript)

**Reply by lvca** [reply]
Forget about paragraphs: words are 90% of the response. I think the most common use case, by far, is to just grab the `transcript` property. So why not provide a new option like `returnWords=true|false` equals to `true` by default, so it would be 100% compatible with the past, but you can pass false and have a much shorter output?
